### PR TITLE
[css-grid-1] Add tests for percentage tracks inside shrinkwrapped width/height.

### PR DESCRIPTION
**Pull request by fantasai**
https://github.com/w3c/csswg-drafts/issues/1921

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21507)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/312242384)

File: css/css-grid/layout-algorithm/grid-percent-cols-filled-shrinkwrap-001.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<link rel="match" href="references/grid-percent-cols-filled-shrinkwrap-001-ref.html">
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="author" title="Elika J. Etemad" href="http://fantasai.inkedblade.net/contact">
+<style>
+    .grid {
+        display: grid;
+        float: left;
+        margin: 1em;
+        grid-template-columns: auto 20% auto;
+        align-content: center;
```
Comment (mrego) [inline css/css-grid/layout-algorithm/grid-percent-cols-filled-shrinkwrap-001.html]: Nit: This is not needed.

File: css/css-grid/layout-algorithm/grid-percent-cols-spanned-shrinkwrap-001.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+<link rel="match" href="references/grid-percent-cols-spanned-shrinkwrap-001-ref.html">
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="author" title="Elika J. Etemad" href="http://fantasai.inkedblade.net/contact">
+<style>
+    .grid {
+        display: grid;
+        float: left;
+        margin: 1em;
+        grid-template-columns: auto 20% auto;
+        align-content: center;
```
Comment (mrego) [inline css/css-grid/layout-algorithm/grid-percent-cols-spanned-shrinkwrap-001.html]: Nit: Ditto.

File: css/css-grid/layout-algorithm/grid-percent-rows-filled-shrinkwrap-001.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Filled Percentage Column, Shrinkwrap Width (via float)</title>
```
Comment (mrego) [inline css/css-grid/layout-algorithm/grid-percent-rows-filled-shrinkwrap-001.html]: This should talk about row and height (not column and width).

File: css/css-grid/layout-algorithm/grid-percent-cols-filled-shrinkwrap-001.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Spanned Percentage Column, Shrinkwrap Width (via float)</title>
```
Comment (mrego) [inline css/css-grid/layout-algorithm/grid-percent-cols-filled-shrinkwrap-001.html]: There's no span in this test.

File: css/css-grid/layout-algorithm/grid-percent-rows-filled-shrinkwrap-001.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="author" title="Elika J. Etemad" href="http://fantasai.inkedblade.net/contact">
+<style>
+    .grid {
+        display: grid;
+        float: left;
+        margin: 1em;
+        grid-template-rows: auto 20% auto;
+        grid-auto-flow: column;
+        justify-content: center;
```
Comment (mrego) [inline css/css-grid/layout-algorithm/grid-percent-rows-filled-shrinkwrap-001.html]: Nit: Again I believe this line is not needed..

File: css/css-grid/layout-algorithm/grid-percent-rows-spanned-shrinkwrap-001.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Spanned Percentage Column, Shrinkwrap Width (via float)</title>
```
Comment (mrego) [inline css/css-grid/layout-algorithm/grid-percent-rows-spanned-shrinkwrap-001.html]: Ditto aout "column" and "width".

File: css/css-grid/layout-algorithm/grid-percent-rows-spanned-shrinkwrap-001.html [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="author" title="Elika J. Etemad" href="http://fantasai.inkedblade.net/contact">
+<style>
+    .grid {
+        display: grid;
+        float: left;
+        margin: 1em;
+        grid-template-rows: auto 20% auto;
+        grid-auto-flow: column;
+        justify-content: center;
```
Comment (mrego) [inline css/css-grid/layout-algorithm/grid-percent-rows-spanned-shrinkwrap-001.html]: Nit: Ditto.

**Review by mrego — APPROVED** [review]
Thanks for the tests, please review the titles as there are some inconsistencies there. :wink:

**Comment by gsnedders** [comment]
Is there any reason this hasn't landed? We should either land it and then file an issue to fix the nits, or fix the nits and land it soon; it lying around in a PR isn't very helpful.

**Comment by mrego** [comment]
@gsnedders no good reason, I just reviewed it and I guess fantasai didn't find time to come back to it.

I'm merging it and I'll send a PR fixing the nits.

**Comment by mrego** [comment]
The nits have been fixed in #12964.

**Pull request merged**